src/Symfony/Component/ExpressionLanguage/ExpressionLanguage.php

```diff
@@ -25,8 +25,20 @@ class ExpressionLanguage
      * @var ParserCacheInterface
      */
     private $cache;
+
+    /**
+     * @var Lexer
+     */
     private $lexer;
+
+    /**
+     * @var Parser
+     */
     private $parser;
+
+    /**
+     * @var Compiler
+     */
```

Comment:
On a more general note, we add just the right amount of phpdoc that really helps. If it does not help, we prefer to not use them.